Sample-01/api-server.js

```diff
@@ -10,10 +10,12 @@ const app = express();
 if (
   !authConfig.domain ||
   !authConfig.authorizationParams.audience ||
-  ["YOUR_API_IDENTIFIER", "{API_IDENTIFIER}"].includes(authConfig.authorizationParams.audience)
+  ['{yourApiIdentifier}', '{API_IDENTIFIER}'].includes(
```

Comment:
Same comment as the react sample app, do we need to keep "{API_IDENTIFIER}" for a reason? I am just wondering why it was added/missing in the react sample app or what its purpose is.

Reply:
I just looked at some python sample apps and saw that they have "{API_IDENTIFIER}" as the default value for API_IDENTIFIER. Please disregard the above comment.